cylc/flow/scheduler_cli.py

```diff
@@ -406,13 +406,7 @@ async def _run(parser, options, reg, is_restart, scheduler):
     # stop cylc stop
     except SchedulerError:
         ret = 1
-    except KeyboardInterrupt as exc:
-        try:
-            await scheduler.shutdown(exc)
-        except Exception as exc2:
-            # In case of exceptions in the shutdown method itself.
-            LOG.exception(exc2)
-            raise exc2 from None
```

Comment:
No need for this anymore as the shutdown is handled in `scheduler.run()` or `scheduler.start_scheduler()`